modules/apps/portal-search-solr8/portal-search-solr8-impl/src/main/java/com/liferay/portal/search/solr8/internal/interceptor/PreemptiveAuthInterceptor.java

```diff
@@ -30,7 +30,9 @@
 /**
  * @author Eric Min
  */
-@Component(service = HttpRequestInterceptor.class)
+@Component(
+	service = {HttpRequestInterceptor.class, PreemptiveAuthInterceptor.class}
```

Comment:
@W2jx We are currently trying to remove the usages to register multiple services, so please do not do this.